src/index.d.ts

```diff
@@ -561,7 +561,7 @@ declare namespace SDKStandardComponents {
         method: RequestMethod
         uri: string
         agent: http.Agent
-        qs?: string
+        qs?: { [key: string]: any }
```

Comment:
it is better to use `unknown` instead `any`
https://stackoverflow.com/questions/51439843/unknown-vs-any

```suggestion
        qs?: { [key: string]: unknown }
```